foodhub_copy_product_fields/models/product_template.py

```diff
@@ -0,0 +1,31 @@
+# SPDX-FileCopyrightText: 2022 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    volume = fields.Float(copy=True)
```

Comment:
a comment explaining that only the `copy` property is changed on existing fields would be nice. at this time, it’s clear, but we don’t know what will happen later. this is a useful information for code migration, for example.